test/scripting/scriptingProvider.spec.js

```diff
@@ -125,3 +125,70 @@ describe('scriptingProvider._tagAndCleanupUtterances', function () {
     expect(actualResult[0].utterances).to.eql(utterances.slice(0, 2))
   })
 })
+
+describe('scriptingProvider.ExpandUtterancesToConvos', function () {
+  it('should build plain convos for utterance', async function () {
+    const scriptingProvider = new ScriptingProvider(DefaultCapabilities)
+    await scriptingProvider.Build()
+    scriptingProvider.AddUtterances({
+      name: 'utt1',
+      utterances: ['TEXT1', 'TEXT2']
+    })
+
+    scriptingProvider.ExpandUtterancesToConvos()
+    assert.equal(scriptingProvider.convos.length, 1)
```

Comment:
this would be more readable assert.lengthOf([1,2,3], 3, 'array has length of 3');